### PR TITLE
Docs: JuiceFS on Docker

### PR DESCRIPTION
**Pull request by yuhr123**

### CODE REVIEW

File: docs/en/juicefs_on_docker.md [hunk 1]
```diff
@@ -1 +1,50 @@
...
+
+### Change the configuration file
+
+Edit the configuration file of FUSE, usually `/etc/fuse.conf`:
+
+```
+$ sudo nano /etc/fuse.conf
+```
+
+Delete the `# ` symbol in front of ``user_allow_ other` in the configuration file, and modify it as follows:
```
Comment (xiaogaozi) [inline docs/en/juicefs_on_docker.md:15]:
```
``user_allow_ other` -> `user_allow_other`
```

File: docs/en/juicefs_on_docker.md [hunk 2]
```diff
@@ -1 +1,50 @@
...
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
+```
+
+### Re-mount JuiceFS
+
+After the `allow_other` of FUSE is enabled, you need to re-mount the JuiceFS file systemd with the `allow_other` option, for example:
+
+```
+$ sudo juicefs mount -d -o allow_other redis://<your-redis-url>:6379/1 /mnt/jfs
```
Comment (xiaogaozi) [inline docs/en/juicefs_on_docker.md:33]: `sudo` means mount with root user, if then no need use `-o allow_other`.

**Review by xiaogaozi — CHANGES_REQUESTED** [review]

**Review by xiaogaozi — APPROVED** [review]

File: docs/zh_cn/juicefs_on_docker.md [hunk 3]
```diff
@@ -1,2 +1,55 @@
 # Docker 使用 JuiceFS
 
+如果你希望使用 JuiceFS 为 Docker 容器提供持久化的存储，你需要先开启 FUSE 的 `user_allow_other` 选项，然后通过  `-o` 选项设置 `allow_other`重新挂载 JuiceFS 文件系统。
```
Comment (davies) [inline docs/zh_cn/juicefs_on_docker.md:3]: user_allow_other is only needed if you are not root.

File: docs/zh_cn/juicefs_on_docker.md [hunk 4]
```diff
@@ -1,2 +1,55 @@
...
+# Allow non-root users to specify the allow_other or allow_root mount options.
+user_allow_other
+```
+
+### 重新挂载 JuiceFS
+
+FUSE 的 `user_allow_other` 启用后，你需要重新挂载 JuiceFS 文件系统，使用 `-o` 选项设置 `allow_other`，例如：
+
+```
+$ juicefs mount -d -o allow_other redis://<your-redis-url>:6379/1 /mnt/jfs
```
Comment (davies) [inline docs/zh_cn/juicefs_on_docker.md:33]: juicefs adds `allow_other` when the current user is root or using `sudo`

Comment (yuhr123) [reply docs/zh_cn/juicefs_on_docker.md:33]: When I mapping a sub-directory in JuiceFS mount point to a Docker container, I got the following output:
`2021-05-19 13:58:51.857830 I | Unimplemented opcode POLL`

FYI: My JuiceFS is composed with MinIO and MySQL, and I have enable user_allow_other in `/etc/fuse.conf`.

The command I am using to mount JuiceFS is:
```shell
$ sudo juicefs mount mysql://juicefs:xxxxxx@(localhost:3306)/juicefs juice-test/
```

And then I create a filebrowser container:
```shell
sudo docker run -d --name fb \
-p 8088:80 \
-v $PWD/fb/root:/srv \
-v $PWD/fb/database.db:/database.db \
filebrowser/filebrowser
```

After the container created, the app in container is running failed, Docker logs:
```
2021/05/19 13:57:46 bolt.Close(): funlock error: function not implemented
2021/05/19 13:57:46 function not implemented
```

But by the same way, I can create the container successful when I use a local directory.

Comment (davies) [reply docs/zh_cn/juicefs_on_docker.md:33]: This one is OK.
```
2021-05-19 13:58:51.857830 I | Unimplemented opcode POLL
```

The SQL implementation does not support flock and fcntl, please use Redis in this case.

**Pull request merged**